src/components/ebay-menu/test/test.browser.js

```diff
@@ -269,10 +269,11 @@ describe('given the menu is in the expanded state with radio items', () => {
 
     describe('when an item\'s inner span is clicked', () => {
         let spy;
-        beforeEach(() => {
+        beforeEach((done) => {
```

Comment:
Why did we have to make the init async? What changed?

Reply:
Not quite sure. These are all cases that were fine in v3, but failed in v4 until I made them async. I'm guessing that for some reason v4 needs the one extra tick from `setTimeout(done)` to fully set up the component.